ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js

```diff
@@ -60,12 +60,47 @@ export function saveToCatalogModalDirective($rootScope, $uibModal, $injector, co
         link: link
     };
 
-    function link($scope, $element) {
+    function link($scope, $element, $compile, controller) {
         $scope.buttonText = $scope.config.label || ($scope.config.itemType ? `Update ${$scope.config.name || $scope.config.symbolicName}` : 'Add to catalog');
 
         $scope.activateModal = () => {
-            // Override callback to update catalog configuration data in other applications
-            $scope.config = (composerOverrides.updateCatalogConfig || (($scope, $element) => $scope.config))($scope, $element);
+            function injectorGet(reference) { return $element.injector().get(reference); }
+            function blueprintService() { return injectorGet('blueprintService'); }
```

Comment:
Instead of using the `$injector`, you can inject `blueprintService` directly in `saveToCatalogModalDirective`